build/charts/antrea-windows/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: antrea-windows
+type: application
+displayName: Antrea
+home: https://antrea.io/
+version: 0.0.0
+appVersion: latest
+kubeVersion: ">= 1.16.0-0"
```

Comment:
This should be `1.19.0-0` now, could you also update the one in Linux Chart.yaml?